### PR TITLE
percobaan review

### PR DESCRIPTION
**Pull request by riyhs**
deskripsi percobaan review
@rygenzx tolong code saya di review

### CODE REVIEW

**Comment by rigenski** [comment]
good @riyhs

File: riyaldi.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # FILE MD RIYALDI
 test 1
 test 2
```
Comment (riyhs) [inline riyaldi.md:3]: test comment

File: riyaldi.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # FILE MD RIYALDI
```
Comment (riyhs) [inline riyaldi.md:1]: comment

File: riyaldi.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
 test 1
```
Comment (rigenski) [inline riyaldi.md:2]: test

**Review by rigenski — COMMENTED** [review]
Testt

**Review by rigenski — APPROVED** [review]
gooddd

**Review by rigenski — APPROVED** [review]
Tests

**Pull request merged**